tests/kafkatest/tests/connect/connect_rest_test.py

```diff
@@ -90,7 +90,8 @@ def test_rest_api(self, connect_protocol, metadata_quorum):
         self.cc.set_configs(lambda node: self.render("connect-distributed.properties", node=node))
         self.cc.set_external_configs(lambda node: self.render("connect-file-external.properties", node=node))
 
-        self.cc.start()
+        self.logger.info("Waiting till Connect REST server is listening")
+        self.cc.start(mode=ConnectServiceBase.STARTUP_MODE_LISTEN)
```

Comment:
I think this is unnecessary for the stabilization fix, and actually weakens the test. Because this is actually creating the connectors in distributed mode, I think it would be smart to wait for the cluster to actually join the cluster.

So we can revert these two lines and leave it as it was.